src/CMakeLists.txt

```
@@ -316,4 +323,4 @@ target_link_libraries(${PROJECT_NAME}.x PUBLIC ${PROJECT_NAME})
 
 ### Export and installation
 
-fesom_export(TARGETS ${PROJECT_NAME} parms fesom.x)
+fesom_export(TARGETS ${PROJECT_NAME} parms fesom.x)
```

```suggestion
fesom_export(TARGETS ${PROJECT_NAME} parms fesom.x)

```
This suggestion is to avoid unnecessary changes